torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1945,10 +1945,10 @@ def hint():
                     msg = f"  {len(error_msgs) + 1}. {msg()}"
                     error_msgs.append(msg)
             if len(error_msgs) > 0:
-                log.warning("Warning only constraints violated %s", warn_msgs)
+                log.info("Warning only constraints violated %s", warn_msgs)
```

Comment:
Wait, this looks like a botched merge, we shouldn't warn here at all, we're going to raise an error!